detection-rules/attachment_any_html_suspicious_javascript_identifiers.yml.yml

```
@@ -0,0 +1,23 @@
+name: "Attachment: Any HTML file with Suspicious JavaScript Identifiers"
+description: |
+  Potential HTML obfuscation attack based on suspicious JavaScript identifiers.
+  Some attackers may use obfuscation techniques such as ROT13 to bypass email security filters.
+  This rule may be expanded to inspect HTML attachments for other suspicious identifiers.
+references:
+  - ""
+type: "rule"
```

```suggestion
type: "rule"
authors:
  - twitter: "Kyle_Parrish_"
    name: "Kyle Parrish"
```

don't forget to give yourself credit :) you could do either one or both of these